src/sdk/model/edge/edge.ts

```diff
@@ -218,32 +220,45 @@ export class Edge extends Entity {
 
   /**
    * Get the Edge firewall
+   * @deprecated This method is deprecated, use getEdgeFirewall() instead
    * @returns {Promise<Firewall>}
    */
-  async getFirewall(): Promise<Firewall> {
+  async getFirewall(): Promise<EdgeFirewall> {
```

Comment:
@fradinni so `getFirewall` returns `EdgeFirewall` and `getEdgeFirewall` returns `Firewall`? Doesn't that seem backwards?

Reply:
still wondering about this @fradinni ?